backend/src/middleware/slugifyMiddleware.spec.js

```diff
@@ -77,7 +77,7 @@ describe('slugify', () => {
   describe('CreateUser', () => {
     const action = async (mutation, params) => {
       return authenticatedClient.request(`mutation {
-        ${mutation}(password: "yo", ${params}) { slug }
+        ${mutation}(password: "yo", email: "123@123.de", ${params}) { slug }
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-02T17:44:08Z" title="Thursday, May 2nd 2019, 7:44:08 pm +02:00">May 2, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -77,7 +77,7 @@ describe('slugify', () => {
   describe('CreateUser', () => {
     const action = async (mutation, params) => {
       return authenticatedClient.request(`mutation {
-        ${mutation}(password: "yo", ${params}) { slug }
+        ${mutation}(password: "yo", email: "123@123.de", ${params}) { slug }
```


@ulfgebhardt do you know a way in GraphQL how to supply those variables inside the parantheses only if their passed to the mutation?
```
mutation($some: TYPE, $variable: TYPE, $here: TYPE) {
  WhateverMutation(... this should be empty if none of the variables above are given) {
    ....
  }
}
```

Reply:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-05-06T10:58:13Z" title="Monday, May 6th 2019, 12:58:13 pm +02:00">May 6, 2019</time>_

---


No clue what you mean